src/main/resources/db/changelog/changes/04-create-users-roles-table.yaml

```diff
@@ -0,0 +1,16 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users-roles-table
+      author: anton4uk
+      changes:
+        - createTable:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: role_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: user_id
+                  type: bigint
```

Comment:
I suggest to add this

                  constraints:
                    nullable: false
                    foreignKeyName: user_foreignKey
                    references: users(id)

to make both of this fields actual foreign keys

Reply:
Great point!